layouts/quickstart/index.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html lang="en">
+
+  {{ partial "head-landing.html" . }}
+
+  <body>
+
+    {{ partial "header.html" . }}
+
+    <!-- <div class="header-view">
+      <div class="usa-grid">
+        <h1><a class="link-textblack" href="/updates/">Updates</a></h1>
+      </div>
+    </div> -->
+
+    <main class="quickstart">
+
+      <div class="usa-content bgcolor-deepblue">
+        <div class="usa-grid">
+          <section class="hero">
+            <h1>Try cloud.gov for free</h1>
+            <p>
+              Follow this quick start guide to create an account and deploy a sample app in your own free sandbox.
+            </p>
+          </section>
+
+
+
+          <!-- <div class="usa-width-two-thirds panel content-text">
+            <header class="post-header">
+              <span class="panel-new-date">{{ .Date.Format "Jan 2, 2006" }}{{ if .Params.author }} • {{ .Params.author }}{{ end }}{{ if .Params.meta }} • {{ .Params.meta }}{{ end }}</span>
+            </header>
+            <h1>{{ .Title }}</h1>
+
+            {{ if .Params.image }}
+            <div class="page--banner" style="background-image: url({{ .Params.image}});"></div>
+            {{ end }}
+
+            {{ .Content }}
+         </div> -->
+
+        </div>
+      </div>
+
+      <div class="usa-content bgcolor-ltbluegrey">
+        <div class="usa-grid">
+          <section class="step first">
+            <h2 id="access">Get trial access and a free sandbox account</h2>
+            <p>
+              Federal employees and contractors with an email ending in .mil, .gov, or .fed.us can obtain a free sandbox account. To register, you’ll need access to your email and a smartphone with Google Authenticator installed.
+            </p>
+            <!-- <form>
+              <label for="email">Federal email address</label>
+              <div class="form-wrapper">
+                <input type="text" name="email" value="">
+                <input type="submit" value="Submit">
+              </div>
+            </form> -->
+            <p>
+              <a href="https://cloud.gov/docs/getting-started/accounts/" class="usa-button">Get access</a>
```

Comment:
Let's link directly to https://account.fr.cloud.gov/signup (see https://gsa-tts.slack.com/archives/C4XLJ1JBV/p1510777124000397)